clusters/lib/nerc-images/application.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: nerc-images
+  labels:
+    nerc.mghpcc.org/sync-policy: common
+spec:
+  destination:
+    name: nerc-ocp-prod
+    namespace: redhat-ods-applications
```

Comment:
I was wondering, do we have to specify a namespace here, or should I leave this out?

Reply:
@computate did we get an answer on this topic yet?

Reply:
The namespace is used for any resources that don't otherwise specify a namespace in their manifests. A value is not *required*, but I would include one to avoid resources accidentally showing up in random namespaces. Just set it to the namespace in which the bulk of your resources will be deployed.

Reply:
I have set the path to `overlays/nerc-ocp-prod` in nerc-images to be more consistent.